lib/node_modules/@stdlib/_tools/scaffold/math-strided-real-typed-unary/data/benchmark/c/benchmark__length__c.txt

```diff
@@ -85,7 +87,7 @@ double tic() {
 *
 * @param a    minimum value
 * @param b    maximum value
-* @return random number
+* @return     random number
 */
 double rand_uniform( double a, double b ) {
```

Comment:
I believe we are missing `static` here and elsewhere, correct?

Reply:
Yes, I've updated it now.